CONTRIBUTING.md

```diff
@@ -112,6 +113,15 @@ We use a branching model that helps us keep track of branches in a logical, cons
 | `fix`            | Non-breaking change which fixes an issue                                |
 | `tests`          | Changes to project unit `tests/` and / or integration `features/` tests |
 
+### Pull request title conventions
+
+Kedro requires that you [squash and merge your pull request commits](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits). In most cases, the [merge message for a squash merge](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#merge-message-for-a-squash-merge) will default to the pull request title; therefore, pull request titles should follow [the rules pertaining to the subject of a great Git commit message](https://chris.beams.io/posts/git-commit/#seven-rules):
+
+* [Limit the pull request title to 50 characters](https://chris.beams.io/posts/git-commit/#limit-50)
+* [Capitalize the pull request title](https://chris.beams.io/posts/git-commit/#capitalize)
+* [Do not end the pull request title with a period](https://chris.beams.io/posts/git-commit/#end)
+* [Use the imperative mood in the pull request title](https://chris.beams.io/posts/git-commit/#imperative)
```

Comment:
```suggestion
* [Use the imperative tense for your pull request title](https://chris.beams.io/posts/git-commit/#imperative)
```

Reply:
I think we should change to "tense" since that's correct. Imperative isn't a 'mood' as far as I know (though it sometimes seems that way!). I'll suggest an alternative for the bullets...

Reply:
```suggestion
For clarity, your pull request title should be descriptive and we ask you to follow some guidelines suggested by [Chris Beams](https://github.com/cbeams) in his post [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/#seven-rules). In particular, for your pull request title, we suggest that you do the following:

* [Limit the length to 50 characters](https://chris.beams.io/posts/git-commit/#limit-50)
* [Capitalise the first letter of the word](https://chris.beams.io/posts/git-commit/#capitalize)
* [Omit the period at the end](https://chris.beams.io/posts/git-commit/#end)
* [Use the imperative tense](https://chris.beams.io/posts/git-commit/#imperative)
```

Reply:
I've taken out the duplication in the bullets. See what you think -- I'm not obsessive about this or anything, so go with what you like the most 😆

Reply:
```suggestion
For clarity, your pull request title should be descriptive and we ask you to follow some guidelines suggested by [Chris Beams](https://github.com/cbeams) in his post [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/#seven-rules). In particular, for your pull request title, we suggest that you:

* [Limit the length to 50 characters](https://chris.beams.io/posts/git-commit/#limit-50)
* [Capitalise the first letter of the first word](https://chris.beams.io/posts/git-commit/#capitalize)
* [Omit the period at the end](https://chris.beams.io/posts/git-commit/#end)
* [Use the imperative tense](https://chris.beams.io/posts/git-commit/#imperative)
```
Minor adjustments. :)